### PR TITLE
Plane: changed flaperon implementation to scaled output

### PR DESCRIPTION
**Pull request by tridge**
This switches from the PWM based to scaled based flaperons in plane. This makes them consistent with the new elevon approach

### CODE REVIEW

File: ArduPlane/servos.cpp [hunk 1]
```diff
@@ -255,29 +255,20 @@ void Plane::channel_function_mixer(SRV_Channel::Aux_servo_function_t func1_in, S
  */
 void Plane::flaperon_update(int8_t flap_percent)
 {
-    if (!SRV_Channels::function_assigned(SRV_Channel::k_flaperon1) ||
-        !SRV_Channels::function_assigned(SRV_Channel::k_flaperon2)) {
+    if (!SRV_Channels::function_assigned(SRV_Channel::k_flaperon_left) &&
+        !SRV_Channels::function_assigned(SRV_Channel::k_flaperon_right)) {
```
Comment (magicrub) [inline ArduPlane/servos.cpp]: This looks like you've fixed a bug? Right?

Comment (tridge) [inline ArduPlane/servos.cpp]: no, both are correct, just different level of optimisation. Previously we only generated flaperons if both outputs were defined. Now we allow you to have just one defined.

**Comment by tridge** [comment]
this has only been flight tested in simulators so far

**Comment by tridge** [comment]
flight tested on Sunday, merging

**Pull request merged**